packages/embeddable/src/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title><%= htmlWebpackPlugin.options.title %></title>
+    <style>
+      body {
+        cursor: default;
+        font-family: "Nunito Sans", sans-serif;
+        margin: 0;
+        padding: 32px;
+      }
+
+      body,
+      body * {
+        box-sizing: border-box;
+      }
+    </style>
+    <script type="text/javascript">
+      window.courierConfig = {
+        clientKey: "ZDRiMTViNmMtODdmZi00OWMzLWI5YTgtYmU5NTc3YTRlM2Y1",
```

Comment:
Should we initialize this without userId and clientKey?

Reply:
you mean, should we support initializing async as well as sync?